posthog/src/main/java/com/posthog/internal/PostHogLocalEvaluationModels.kt

```diff
@@ -0,0 +1,191 @@
+package com.posthog.internal
+
+import com.google.gson.annotations.SerializedName
+import com.posthog.PostHogInternal
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement
+
+/**
+ * Response from /api/feature_flag/local_evaluation/
+ */
+@PostHogInternal
+public data class LocalEvaluationResponse(
+    val flags: List<FlagDefinition>?,
+    @SerializedName("group_type_mapping")
+    val groupTypeMapping: Map<String, String>?,
+    val cohorts: Map<String, PropertyGroup>?,
+)
+
+/**
+ * Complete feature flag definition for local evaluation
+ */
+@IgnoreJRERequirement
+@PostHogInternal
+public data class FlagDefinition(
+    val id: Int,
+    val name: String,
+    val key: String,
+    val active: Boolean,
+    val filters: FlagFilters,
+    val version: Int,
+)
```

Comment:
@marandaneto Is this okay to annotate with `@IgnoreJRERequirement` to suppress these Animal Sniffer errors?
```
[Undefined reference | android-api-level-21-5.0.1_r2] com.posthog.internal.(PostHogLocalEvaluationModels.kt:1)
  >> int Integer.hashCode(int)

[Undefined reference | android-api-level-21-5.0.1_r2] com.posthog.internal.(PostHogLocalEvaluationModels.kt:1)
  >> int Double.hashCode(double)
```

According to [gummy-bears docs](https://github.com/open-toast/gummy-bears#android), this is incorrectly flagged as being added in API 24 despite working in API 19

Reply:
Thanks! I switched to regular classes to avoid generating those methods and dropped the `@IgnoreJRERequirement` annotations.